charts/portworx/v1.0.0/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: portworx
-appVersion: 1.0.0
-version: 1.0.0
+appVersion: 1.0.1
+version: 1.0.1
```

Comment:
the chart will need to move to  a new  v1.0.1 folder. Current changes are in v1.0.0 folder